src/ecdsa/test_pyecdsa.py

```diff
@@ -1203,6 +1203,44 @@ def do_test_to_openssl(self, curve, hash_name="SHA1"):
         )
 
 
+class TooSmallCurve(unittest.TestCase):
+    @pytest.mark.skipif("prime192v1" not in OPENSSL_SUPPORTED_CURVES,
```

Comment:
`OPENSSL_SUPPORTED_CURVES` is defined on line 849 as a class variable of `OpenSSL` class, so it's not available in this scope

Reply:
Yep, I originally developed this inside the `OpenSSL` test case class, and then moved it out at the last minute. I'll fix that.